featurebyte/api/feature.py

```diff
@@ -711,7 +718,34 @@ def save(
         ... )["InvoiceAmountAvg_60days"]
         >>> invoice_amount_avg_60days.save()  # doctest: +SKIP
         """
-        super().save(conflict_resolution=conflict_resolution, _id=_id)
+        sdk_execution_mode = os.environ.get("SDK_EXECUTION_MODE")
```

Comment:
Would it be better to make the name more specific, e.g. `FEATUREBYTE_SDK_EXECUTION_MODE`?